README.md

```diff
@@ -1,22 +1,28 @@
 # Uber Rides Java SDK (Beta) [![Build Status](https://travis-ci.org/uber/rides-java-sdk.svg?branch=master)](https://travis-ci.org/uber/rides-java-sdk)
 This SDK helps your Java App make HTTP requests to the Uber Rides API.
 
+## Prerequisites
+- Dependency manager: Gradle or Maven
```

Comment:
One thing I liked from the okhttp example I linked, is that they reference the latest jar from artifactory as a link. We can do that and actually remove the need for a dep manager for the users with that limitation.